### PR TITLE
Feature/product detail image scrolling

### PR DESCRIPTION
**Pull request by nbradbury**
Adds a parallax scroll effect to the product image in product detail.

ezgif-3-1c2024956a7c

### CODE REVIEW

**Comment by AmandaRiu** [comment]
@nbradbury  I'm still reviewing the code, but noticed something that could be handled in the interim. We talked about making the product image clickable to view the whole image, and I still think that's a great idea. Do you think we should add it in this PR?

Also, I noticed the image height calculation needs adjustment in landscape mode. It just cuts off too much of the image. This is portrait:



and this is landscape...notice the difference:

**Comment by nbradbury** [comment]
> We talked about making the product image clickable to view the whole image, and I still think that's a great idea. Do you think we should add it in this PR?

I think that should be a separate PR given that it'll add a non-trivial amount of code.

> Also, I noticed the image height calculation needs adjustment in landscape mode.

Oof, that does look bad. I'll fix it shortly.

File: WooCommerce/src/main/res/layout/skeleton_product_detail.xml [hunk 1]
```diff
@@ -7,8 +7,9 @@
     android:orientation="vertical">
 
     <View
+        android:id="@+id/productImage_Skeleton"
         android:layout_width="match_parent"
-        android:layout_height="@dimen/product_detail_image_height"
+        android:layout_height="224dp"
```
Comment (AmandaRiu) [inline WooCommerce/src/main/res/layout/skeleton_product_detail.xml]: ❓ Why was this removed from the dimens pool?

Comment (nbradbury) [reply WooCommerce/src/main/res/layout/skeleton_product_detail.xml]: The image height is set programmatically [here](https://github.com/woocommerce/woocommerce-android/pull/927/files#diff-d8f98fdf3c59e83ae9751fb4340f93bbR73) so the dimen was unused.

Comment (AmandaRiu) [reply WooCommerce/src/main/res/layout/skeleton_product_detail.xml]: ah I see.

**Comment by AmandaRiu** [comment]
@nbradbury is it also possible to change the close icon from an "x" to a back arrow to match the order detail view so it feels a little more seamless?

**Comment by nbradbury** [comment]
> is it also possible to change the close icon from an "x" to a back arrow to match the order detail view so it feels a little more seamless?

That was part of @kyleaparker's design. Personally, I like it, and it's the same as we do for media detail in WPAndroid.

**Comment by nbradbury** [comment]
@AmandaRiu In a096533 I changed the image to be a larger % of screen height when in landscape. Ready for round two!
